### DISCUSSION TITLE
Feature to add voice chat recordings

### DISCUSSION THREAD

**Discussion by Dying-Ducks** [akiver/cs-demo-manager#669] [Ideas]
For now, in the 2D Viewer, there is a volume setting which controls how loud the moans of killed players or the radio massage "FLASHBANG" is, when someone throws a nade.

This audio is surprisingly not helpful or useful at all.

When my team plays an official, we record the team speak session, to work on our communication.
It would be nice if one would be able to add this team speak recording to the 2D viewer, and that its automatically sync when jumping within the demo.

So that the current audio is replaced, and we don't hear "FLASHBANG" but our actually calls in that situation. This would be a great feature.

**Comment by akiver** [comment]
Available as of [v3.14.0](https://cs-demo-manager.com/changelog#v3140).
